### PR TITLE
fix: allow empty json repodata

### PR DESCRIPTION
**Pull request by wolfv**
I didn't benchmark this - but I don't think it should have a huge impact. 

LMK what you think, Mr. Bas!

### CODE REVIEW

File: crates/rattler_shell/src/activation.rs [hunk 1]
```diff
@@ -261,7 +261,7 @@ fn collect_env_vars(prefix: &Path) -> Result<IndexMap<String, String>, Activatio
 /// # Returns
 ///
 /// A vector of path entries
-fn prefix_path_entries(prefix: &Path, platform: &Platform) -> Vec<PathBuf> {
+pub fn prefix_path_entries(prefix: &Path, platform: &Platform) -> Vec<PathBuf> {
```
Comment (baszalmstra) [inline crates/rattler_shell/src/activation.rs]: This change seems unrelated

```suggestion
fn prefix_path_entries(prefix: &Path, platform: &Platform) -> Vec<PathBuf> {
```

Comment (wolfv) [reply crates/rattler_shell/src/activation.rs]: yep, it's gone!
